src/views/OrderDetail.vue

```diff
@@ -658,10 +658,16 @@ export default defineComponent({
           },{
             text: header,
             handler: async () => {
+              alert.dismiss();
+              emitter.emit("presentLoader", {message: "Loading...", backdropDismiss: false});
```

Comment:
Adding a loader here will lead to have two loaders on the screens one from here and another from the actions.